doc/architecture/decisions/0004-reference-data-sync.md

```diff
@@ -296,8 +308,14 @@ e.g. **/incentive/prison-levels/MDI** returns
 ]
 ```
 
+#### Get the details of all active incentive levels for a specified prison
+This response is the same as above but includes `"active": false`
+
+`GET /incentive/prison-levels/{prisondId}?with-inactive=true`
```

Comment:
I like that by default only active are returned but it's possible to request all of them when needed 👍

Reply:
i would have preferred to hide inactive ones but then we cannot tell nomis something has been deactivated :(